### PR TITLE
add title-short to all SBL variants

### PR DESCRIPTION
**Pull request by POBrien333**
Noticed this a few times already as I always search for SBL and can only find 1 style on the repository.

### CODE REVIEW

File: society-of-biblical-literature-1st-edition-fullnote-bibliography.csl [hunk 1]
```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
-<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US" page-range-format="chicago">
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="chicago" default-locale="en-US">
   <info>
     <title>Society of Biblical Literature 1st edition (full note)</title>
+    <title-short>SBL</title-short>
```
Comment (denismaier) [inline society-of-biblical-literature-1st-edition-fullnote-bibliography.csl:5]: Wouldn't it make sense to add the edition? Maybe change it to "SBL (1st edition)"

File: society-of-biblical-literature-fullnote-bibliography.csl [hunk 2]
```diff
@@ -2,6 +2,7 @@
 <style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="chicago" default-locale="en-US">
   <info>
     <title>Society of Biblical Literature 2nd edition (full note)</title>
+    <title-short>SBL</title-short>
```
Comment (denismaier) [inline society-of-biblical-literature-fullnote-bibliography.csl:5]: Wouldn't it make sense to add the edition? Maybe change it to "SBL (2nd edition)"

**Review by denismaier — CHANGES_REQUESTED** [review]

File: society-of-biblical-literature-1st-edition-fullnote-bibliography.csl (hunk 1 above)
Comment (adam3smith) [reply society-of-biblical-literature-1st-edition-fullnote-bibliography.csl:5]: Agree - while the main function here is findability (which this PR provides) it seems weird to have the exact same short title for multiple styles. @denismaier 's suggestion follows what we do for APA

**Comment by POBrien333** [comment]
Thanks for the comments. I've addressed this in the PR.

**Review by denismaier — APPROVED** [review]

**Pull request merged**